services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None) -> QuerySet:

```

Comment:
Specify inner type of QuerySet.
```suggestion
        actors_ids: list[int] = None) -> QuerySet[Movie]:

```